inspirehep/modules/records/jsonschemas/records/authors.json

```diff
@@ -0,0 +1,274 @@
+{
```

Comment:
Here's one case in which an automatic rebase gets it wrong : )

If you are doing schema modifications, you have to submit a PR to https://github.com/inspirehep/inspire-schemas.